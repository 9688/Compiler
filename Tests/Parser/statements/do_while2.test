do{
   x++;
}while(1);