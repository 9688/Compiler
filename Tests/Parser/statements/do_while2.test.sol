
DO WHILE
     list
          1
     BLOCK
          list
               "@++"
                    x

