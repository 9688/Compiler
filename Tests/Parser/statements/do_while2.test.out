юдд''
   юдд'do'
      цдд'1'
      юдд'block'
         юдд'@++'
            юдд'x'

